pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: sass
-version: 1.38.0
+version: 1.38.1
```

Comment:
Always add a CHANGELOG entry, even if the changes aren't visible here. Otherwise people are liable to get confused, and I think it'll break the release process too.